thorlcr/thorutil/thmem.cpp

```diff
@@ -720,6 +723,12 @@ void CThorExpandingRowArray::setup(IThorRowInterfaces *_rowIf, EmptyRowSemantics
         ReleaseThorRow(stableTable);
         stableTable = NULL;
     }
+    if (stableSort_none != stableSort)
+    {
+        StringBuffer algorithm;
+        activity.getOpt(THOROPT_SORT_ALGORITHM, algorithm);
```

Comment:
>this is the best place I could find to add the option, but it isn't very satisfactory because setup() is called from various places - e.g, swapping arrays and resetting them. I suspect this class would benefit from review and refactoring (I added it to the offsite jira), but not now.

I think it's okay as it is, but it possibly related to a JIRA u opened re. efficiency of getOptBool calls, we could cache these option getOptX lookups, mitigating the concern.